### PR TITLE
SOT-131: Add Folding Scheme for Committed Relaxed R1CS

### PR DESCRIPTION
**Pull request by VanhGer**
# Check List

- [x] Did you check your grammar / reader proof your document using ChatGPT?
- [x] Don't forget to squash commits into meaningful chunks before merging
- [x] After merging this, please update your task status!
- [x] Did you verify that your math formulas are rendered correctly?

### CODE REVIEW

File: docs/nova.md
Comment (zk-steve) [inline docs/nova.md:1]: fix broken image in this https://github.com/sota-zk-labs/zkp-documents/blob/sot-131/terms/r1cs.md by adding a new line below the image's link

File: docs/nova.md [hunk 1]
```diff
@@ -17,7 +17,56 @@ Read the definition of Folding scheme [here](../terms/folding_scheme.md)
 
 Read the definition of R1CS, Relaxed R1CS and Committed Relaxed R1CS [here](../terms/r1cs.md)
 
+## Folding Scheme for Committed Relaxed R1CS
 
+For Committed Relaxed R1CS, we uses succinct and hiding additively
```
Comment (zk-steve) [inline docs/nova.md:22]:
```suggestion
For Committed Relaxed R1CS, we use succinct and hiding additively 
```

File: docs/nova.md [hunk 2]
```diff
@@ -1,24 +1,68 @@
...
+  - $r_E \leftarrow r_{E_1} + r \cdot r_T + r^2 \cdot r_{E_2}$
+  - $W \leftarrow W_1 + r \cdot W_2$
+  - $r_W \leftarrow r_{W_1} + r \cdot r_{W_2}$
 
+Then, the prover (P) can claim his correct folding by proving that $\bar{W}$ and $\bar{E}$ are the commitments of $W$ and $E$.
 
+You should see the depiction below:
+![Folding Scheme for R1CS](attachments/folding_scheme_for_r1cs.png)
 
+We can make this protocol non-interactive via [Fiat-Shamir](../terms/fiat_shamir.md). Let $\rho$ denote a random oracle, then the
```
Comment (zk-steve) [inline docs/nova.md:65]: link the random oracle

**Review by zk-steve — CHANGES_REQUESTED** [review]
Few things left then it's good to go!

**Review by zk-steve — APPROVED** [review]

**Review by ktokto313 — APPROVED** [review]
chat a

**Pull request merged**